### PR TITLE
chore: fix SonarQube issues

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
Fixed code smells reported by Sonarcloud and Sonarqube

## Links
n/a

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

File: Sources/RInAppMessaging/Models/Trigger.swift [hunk 1]
```diff
@@ -21,23 +21,30 @@ internal struct Trigger: Codable, Equatable {
...
+        case name
+        case value
+        case type
+        case operatorType = "operator"
+    }
+
     let name: String
     let value: String
     let type: AttributeType
-    let `operator`: AttributeOperator
```
Comment (malayli) [inline Sources/RInAppMessaging/Models/Trigger.swift:27]: nit: SonarQube does not accept `operator`?

File: Sources/RInAppMessaging/TooltipDispatcher.swift [hunk 2]
```diff
@@ -113,18 +113,22 @@ internal class TooltipDispatcher: TooltipDispatcherType, ViewListenerObserver {
...
-                        self.queuedTooltips.remove(tooltip)
-                    }
+                    self.handleDisplayCompletion(cancelled: cancelled, tooltip: tooltip)
                 }
             )
             waitForImageDispatchGroup.leave()
         }
     }
 
+    private func handleDisplayCompletion(cancelled: Bool, tooltip: Campaign) {
```
Comment (malayli) [inline Sources/RInAppMessaging/TooltipDispatcher.swift:123]: nit: This function was required by SonarQube?

**Review by malayli — APPROVED** [review]

File: Sources/RInAppMessaging/Models/Trigger.swift (hunk 1 above)
Comment (imaginaris) [reply Sources/RInAppMessaging/Models/Trigger.swift:27]: does not accept names in ` `` `. Anil asked me to change the name if possible (I wanted to make an exception)

File: Sources/RInAppMessaging/TooltipDispatcher.swift (hunk 2 above)
Comment (imaginaris) [reply Sources/RInAppMessaging/TooltipDispatcher.swift:123]: Kind of. I created it to fix complexity (nested closure levels) issue raised by SonarQube (and SonarCloud)

**Pull request merged**